### PR TITLE
CI: Deploy to maven central and pypi from travis.

### PR DESCRIPTION
**Pull request by charliestrawn-wf**
@Workiva/messaging-pp this will trigger deploys to maven central and pypi when a PR is merged into master

cage

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 0</summary>

</details>

File: .travis.yml [hunk 1]
```diff
@@ -15,26 +14,45 @@ before_install:
...
+  - cd $TRAVIS_BUILD_DIR && ./scripts/travis/before-deploy-java.sh
+  - cd $TRAVIS_BUILD_DIR && ./scripts/travis/deploy-java.sh
+  - cd $TRAVIS_BUILD_DIR && ./scripts/travis/deploy-python.sh
 
 notifications:
   email: false
+
+env:
+  global:
+  - secure: QLfcf8lFmF67UAAr/Du7H2KWGV7yeSpueoZC2B9X9Fq8XH5Iv+CPR14miw2YWQjCrlGEo2WeMrUIo0Auz8xQsKCrQe69o7+mwpz79Z60RbzMqRSSUZgnnctwcNQLOsOmfZfElC7DFnL0r/dESsKZ9oMbsTFehQMtSL0dkQGEAiOP0IiAhwdx6a82D8pJemCX66a+e/I/lLZVrypR3HFmOyDNAs6Avpau0eUb28rHaV3iJTK4QvQm/U3ybwdXa0Bt5OacBbjVd9qZKy1dLeMABnhKu/e5Ue1q754pulBnovLItSZ1lTJBNCx64wxpMj/aB+msxDSUUr0sDHiET14Mu3pZS8WzitU2eXRQ84NEt77Ui5OjmLZuHnNjaLy5u0kpNGnL1zmirjFI6MqxOdehdeeAKYS//AZJ/lUe8vxVxxWMqNUtgKIZdl/dHjq4Y163aKpDBsbsrxohq3TiuJiZxIZQ6J8rgULymRfh9Gx83FJAHk4SSwSljrgTqYFmqi+eoPZ/BdxFka2WkxJULb5D9Zv91gw/fNxt1Fy3UtsSaWmpJBFpOO/C6nzqYvu+9aOs3S7pUgm6LeYwLIpsFu6Lw9NAfoYGvx+CvQNDEKlowlBnN6uyfjpU4QEWqZWDgoTUp3bw3NLk8KCSGhwTI8PzGJcs2WS4L4yJsysP98+5D8k=
```
Comment (brianshannan-wf) [inline .travis.yml]: Should probably list what the environment variable name is so if we have to change one of these in the future it's not overly difficult

Comment (charliestrawn-wf) [inline .travis.yml]: isn't it more secure though if even we don't know which one is which?

Comment (charliestrawn-wf) [inline .travis.yml]: we'll update these after we confirm everything works publishing artifacts

**Comment by brianshannan-wf** [comment]
+1

**Pull request merged**